File: loops.py
""" 
    Loops
"""

""" 

for value in values:
    sentences

while condition:
    sentences


"""
start = 1 # default = 0 -> optional
stop = 11
step = 1 # default = 1 -> optional

for i in range(start, stop, step):
    print(i)

nombres = ["Hugo", "Paco", "Luis"]

for i in range(len(nombres)):
    print(nombres[i])

for nombre in nombres:
    print(nombre)

i = 0
while i < len(nombres):
    print(nombres[i])
    i = i + 1

File: loops.js
/* 
    Loops 
*/

/* 

for (inicializador; condition; increment){
    sentences
}

for(index in values){
    sentences
}

for(value of values){
    sentences
}

while(condition){
    sentences
}

do {
    sentences
} while (condition)

*/

for(let i = 1; i <= 10; i++){
    console.log(i);
}

let nombres = ["Hugo", "Pago", "Luis"];

for(let i = 0; i < nombres.length; i++){
    console.log(nombres[i]);
}

for(let index in nombres){
    console.log(nombres[index])
}

for(let nombre of nombres){
    console.log(nombre)
}

let i = 0;
while(i < nombres.length){
    console.log(nombres[i]);
    i++;
}